pages/common/crictl.md

```diff
@@ -0,0 +1,36 @@
+# crictl
+
+> Command-line for CRI-compatible container runtimes.
+> More information: <https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md>.
+
+- List all kubernetes pods(Ready and NotReady):
+
+`crictl pods`
+
+- List all containers (Running and Exited):
+
+`crictl ps --all`
+
+- List all images:
+
+`crictl images`
+
+- Display the status of one or more containers:
+
+`crictl inspect {{container_id}}`
+
+- Open a shell inside a running container:
+
+`crictl exec -it {{container_id}} {{sh}}`
+
+- Pull an image from a registry:
+
+`crictl pull {{image:tag}}`
+
+- Fetch and follow the logs of a container:
+
+`crictl logs -f {{container_id}}`
+
+- Remove one or more images:
```

Comment:
This command can't remove multiple in `docker` :thinking:

Reply:
> This command can't remove multiple in `docker` 🤔

@navarroaxel 
When I executed this command in `docker`, it helped me to delete multiple images.
<img width="525" alt="image" src="https://user-images.githubusercontent.com/53329052/175806868-e261cc51-5964-4fb7-a1a2-ace1ce242d77.png">

Reply:
Oh, I remember this doesn't work. anyway!